bot/utils/scheduling.py

```diff
@@ -161,9 +161,21 @@ def _task_done_callback(self, task_id: t.Hashable, done_task: asyncio.Task) -> N
                 self._log.error(f"Error in task #{task_id} {id(done_task)}!", exc_info=exception)
 
 
-def create_task(coro: t.Awaitable, *suppressed_exceptions: t.Type[Exception], **kwargs) -> asyncio.Task:
-    """Wrapper for `asyncio.create_task` which logs exceptions raised in the task."""
-    task = asyncio.create_task(coro, **kwargs)
+def create_task(
+        coro: t.Awaitable,
+        *suppressed_exceptions: t.Type[Exception],
```

Comment:
I've asked this in <https://github.com/python-discord/bot/pull/1471#discussion_r606778142> but the review didn't get addressed; thoughts on replacing this with an optional kwarg instead of a varag? When calling the function, the purpose of the passed in exceptions isn't immediately clear.

Reply:
I agree with you about the clarity. On the other hand, it's kind of annoying to write the kwarg and wrap everything in a tuple/list, especially when there's only one Exception. I'll say this: as far as I am concerned, if you want to change it, you can. If not, that's fine too.

Reply:
I went ahead and moved it to a kwarg; I'm not too concerned about the small ease of use difference as it's not used that commonly